Specs/Number/Swedish/NumberModel.json

```diff
@@ -882,22 +866,6 @@
       }
     ]
   },
-  {
-    "Input": "femtio-två",
```

Comment:
Same.

Reply:
This case as well as "femtio - två" is a very strange construct. This is not how numbers are written in swedish. I could stretch the recognizer to support "femtio två" (which is also not correct since swedish numbers are written as compound expressions). Should we really encourage completely erroneous grammar?

Reply:
Feel free to remove cases if they really would never show up in practice.
If you can add a list of cases removed with reasons, that helps a lot. Another option is to re-writte really incorrect sentences into new cases, if relevant.
Thanks!

Reply:
Ok. Is there any reason we'd like to support completely erroneous constructs? In an ideal world the recognizer would understand the language in question - not all kind of wild constructs (that might be completeley correct in another language). Supporting non-language specific grammar, in my mind, increases the risk of malinterpretations of the text that is processed. Or, am I missing the point? :)

Reply:
Clearly erroneous cases should not be supported. But malformed cases that may be common-place should, if they don't interfere with regular cases. For example, common typos or very informal language/format, or formats that may not be native that are somewhat commonly used by non-native speakers. 
This is common in English, for example, as people format numbers/datetimes in different styles in international contexts.
Anything "completetly erroneus" or "wild" can be removed. :-)